software/applications/HelloIOIO/build.gradle

```diff
@@ -6,11 +6,26 @@ android {
         minSdkVersion 14
         targetSdkVersion 30
     }
+
+    flavorDimensions "integration"
+    productFlavors {
+        code {
+            dimension "integration"
+        }
+        aar {
+            dimension "integration"
+        }
+    }
+
 }
 
 dependencies {
-    implementation 'com.github.ytai.ioio:IOIOLibAndroid:5.07'
-    implementation 'com.github.ytai.ioio:IOIOLibAndroidBluetooth:5.07'
-    implementation 'com.github.ytai.ioio:IOIOLibAndroidAccessory:5.07'
-    implementation 'com.github.ytai.ioio:IOIOLibAndroidDevice:5.07'
+    codeImplementation project(':IOIOLibAndroidBluetooth')
+    codeImplementation project(':IOIOLibAndroidAccessory')
+    codeImplementation project(':IOIOLibAndroidDevice')
+
+    aarImplementation 'com.github.ytai.ioio:IOIOLibAndroid:5.07' // the old artifacts still need it
```

Comment:
This call as well.

Reply:
It's an `Implementation` of flavor `aar`

Reply:
If you have flavor `aar`  you use `aarImplementation`
With flavor `code`  you use `codeImplementation`